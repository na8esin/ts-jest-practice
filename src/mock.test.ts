it('mocks a function', () => {
  const myMockFn = jest.fn(cb => cb(null, true));

  myMockFn((err, val) => console.log(val));

  expect(myMockFn).toMatchSnapshot();
});
