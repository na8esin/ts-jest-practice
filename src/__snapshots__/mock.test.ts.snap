// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
[MockFunction] {
  "calls": [
    [
      [Function],
    ],
  ],
  "results": [
    {
      "type": "return",
      "value": undefined,
    },
  ],
}
`;

exports[`mocks a function 1`] = `
[MockFunction] {
  "calls": [
    [
      [Function],
    ],
  ],
  "results": [
    {
      "type": "return",
      "value": undefined,
    },
  ],
}
`;
